public/app/features/dashboard/components/GenAI/utils.ts

```diff
@@ -62,7 +62,7 @@ export function getDashboardChanges(dashboard: DashboardModel): {
 export async function isLLMPluginEnabled() {
   // Check if the LLM plugin is enabled.
   // If not, we won't be able to make requests, so return early.
-  return llms.openai.enabled().then((response) => response.ok);
+  return llms.openai.health().then((response) => response.ok);
```

Comment:
Yep, this necessary due to https://github.com/grafana/grafana-experimental/pull/96/files#diff-4ef19306b5076b03914b38b942e939f15e2b640915835e2457e8361adc13df52R350. 

This enabled->health transition should be fine here. Since you're not using any vectordb functionality, your current checking code is sufficient.